packages/stream_chat/lib/src/client/channel.dart

```diff
@@ -1773,9 +1773,8 @@ class ChannelClientState {
   List<Message> get messages => _channelState.messages;
 
   /// Channel message list as a stream.
-  Stream<List<Message>> get messagesStream => channelStateStream
-      .map((cs) => cs.messages)
-      .distinct(const ListEquality().equals);
```

Comment:
@imtoori why we did this change?

Reply:
there was a problem with ephemeral messages not updating related to this